### PR TITLE
Use new round sliders for light card

### PR DESCRIPTION
**Pull request by thomasloven**
Skärmavbild 2019-09-05 kl  19 46 48

### CODE REVIEW

File: src/panels/lovelace/cards/hui-light-card.ts [hunk 1]
```diff
@@ -57,8 +42,6 @@ export class HuiLightCard extends LitElement implements LovelaceCard {
 
   @property() private _roundSliderStyle?: TemplateResult;
```
Comment (bramkragten) [inline src/panels/lovelace/cards/hui-light-card.ts]: Not used anymore?

Comment (thomasloven) [reply src/panels/lovelace/cards/hui-light-card.ts]: Correct.

File: yarn.lock [hunk 2]
```diff
@@ -8588,6 +8595,13 @@ lit-element@^2.0.1, lit-element@^2.2.0:
   dependencies:
     lit-html "^1.0.0"
 
+lit-element@^2.2.1:
```
Comment (iantrich) [inline yarn.lock]: We should probably just upgrade our lit-element to 2.2.1 so we are carrying two versions

File: src/panels/lovelace/cards/hui-light-card.ts [hunk 3]
```diff
@@ -107,7 +90,13 @@ export class HuiLightCard extends LitElement implements LovelaceCard {
           class="more-info"
           @click="${this._handleMoreInfo}"
         ></paper-icon-button>
-        <div id="light"></div>
+        <div id="light">
```
Comment (iantrich) [inline src/panels/lovelace/cards/hui-light-card.ts]:
```suggestion
        <div id="light">
          ${supportsFeature(stateObj, SUPPORT_BRIGHTNESS) ? html`
          <round-slider
            .value=${brightness}
            @value-changing=${this._dragEvent}
            @value-changed=${this._setBrightness}
          ></round-slider>` : ""}
        </div>
```

Closes https://github.com/home-assistant/home-assistant-polymer/issues/3542

File: package.json [hunk 4]
```diff
@@ -64,6 +64,7 @@
     "@polymer/paper-toggle-button": "^3.0.1",
     "@polymer/paper-tooltip": "^3.0.1",
     "@polymer/polymer": "3.1.0",
+    "@thomasloven/round-slider": "^0.2.2",
```
Comment (balloob) [inline package.json]: 🎉 🎉 🎉

File: src/panels/lovelace/cards/hui-light-card.ts (hunk 3 above)
Comment (balloob) [reply src/panels/lovelace/cards/hui-light-card.ts]: Without the slider, how will the card remain any height / be controllable? Probably requires a bit more thought and so better fit for another PR.

File: yarn.lock (hunk 2 above)
Comment (balloob) [reply yarn.lock]: Yes

**Comment by iantrich** [comment]
Partial fix for https://github.com/home-assistant/home-assistant-polymer/issues/3622

File: yarn.lock (hunk 2 above)
Comment (thomasloven) [reply yarn.lock]: I should be able to make the slider accept lower versions of lit-element too, right?

File: package.json (hunk 4 above)
Comment (thomasloven) [reply package.json]: 🎉

File: yarn.lock (hunk 2 above)
Comment (bramkragten) [reply yarn.lock]: Updated lit-element to 2.2.1 in https://github.com/home-assistant/home-assistant-polymer/pull/3614

File: src/panels/lovelace/cards/hui-light-card.ts [hunk 5]
```diff
@@ -7,6 +7,7 @@ import {
   customElement,
 } from "lit-element";
 import "@polymer/paper-icon-button/paper-icon-button";
+import "@thomasloven/round-slider";
```
Comment (bramkragten) [inline src/panels/lovelace/cards/hui-light-card.ts]: Should this be dynamically imported?

File: yarn.lock (hunk 2 above)
Comment (balloob) [reply yarn.lock]: @thomasloven yes, you can mark it as a peer dependency in your `package.json`, that way you tell people: bring your own lit-element.

File: src/panels/lovelace/cards/hui-light-card.ts (hunk 5 above)
Comment (balloob) [reply src/panels/lovelace/cards/hui-light-card.ts]: no. It's tiny.

Comment (balloob) [reply src/panels/lovelace/cards/hui-light-card.ts]: The old one was huge as it required all of jQuery to be added too.

Comment (iantrich) [reply src/panels/lovelace/cards/hui-light-card.ts]: yeah, a massive improvement, I'm sure

**Comment by iantrich** [comment]
So we're not waiting for anything else on this, are we?

**Review by balloob — APPROVED** [review]

**Comment by iantrich** [comment]
this doesn't appear to be part of a release? did it get missed?

**Pull request merged**